samples/dashboards/demo/sync-extremes/demo.details

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Change the directory name to `samples/dashboards/demos/sync-extremes/`

demo -> demos

Reply:
Standard for demo-page directory is `demo` (demos from this directory should show up on the [demo page](https://www.highcharts.com/demo), not `demos`